cartography/intel/aws/ec2/tgw.py

```diff
@@ -0,0 +1,223 @@
+import logging
+
+from .util import get_botocore_config
+from cartography.util import aws_handle_regions
+from cartography.util import run_cleanup_job
+from cartography.util import timeit
+
+
+logger = logging.getLogger(__name__)
+
+
+@timeit
+@aws_handle_regions
+def get_transit_gateways(boto3_session, region):
+    client = boto3_session.client('ec2', region_name=region, config=get_botocore_config())
+    return client.describe_transit_gateways()["TransitGateways"]
+
+
+@timeit
+@aws_handle_regions
+def get_tgw_attachments(boto3_session, region):
+    client = boto3_session.client('ec2', region_name=region, config=get_botocore_config())
+    paginator = client.get_paginator('describe_transit_gateway_attachments')
+    tgw_attachments = []
+    for page in paginator.paginate():
+        tgw_attachments.extend(page['TransitGatewayAttachments'])
+    return tgw_attachments
+
+
+@timeit
+@aws_handle_regions
+def get_tgw_vpc_attachments(boto3_session, region):
+    client = boto3_session.client('ec2', region_name=region, config=get_botocore_config())
+    paginator = client.get_paginator('describe_transit_gateway_vpc_attachments')
+    tgw_vpc_attachments = []
+    for page in paginator.paginate():
+        tgw_vpc_attachments.extend(page['TransitGatewayVpcAttachments'])
+    return tgw_vpc_attachments
+
+
+@timeit
+def load_transit_gateways(neo4j_session, data, region, current_aws_account_id, aws_update_tag):
+    ingest_transit_gateway = """
+    MERGE (ownerAccount:AWSAccount {id: {OwnerId}})
+    ON CREATE SET ownerAccount.firstseen = timestamp(), ownerAccount.foreign = true
+    SET ownerAccount.lastupdated = {aws_update_tag}
+
+    MERGE (tgw:AWSTransitGateway {arn: {ARN}})
```

Comment:
Can we change this to
```
MERGE (tgw:AWSTransitGateway {id: {ARN}})  // see note 1
ON CREATE SET tgw.arn = {ARN},             // see note 2
...
SET tgw.tgw_id = {TgwId}                   // see note 3
```
?

Note 1: I realize cartography is inconsistent in this handling of IDs, but we're trying to standardize now on using `id` as the main identifier.

Note 2: tgw.arn will be equivalent to tgw.id. This is so that the user will use `id` as the unique identifier and know that `arn` was used to derive that identifier.

Note 3: It is also helpful to include an asset identifier like TGW ID separately

Reply:
Implemented in bebf6a0